oio/cli/admin/cluster.py

```diff
@@ -61,3 +63,141 @@ def take_action(self, parsed_args):
         columns = ('Type', 'Id', 'Volume', 'Location', 'Slots', 'Up', 'Score')
         result_gen = (r for r in results)
         return columns, result_gen
+
+
+class ClusterLocalList(lister.Lister):
+    """Cluster local list"""
+
+    log = logging.getLogger(__name__ + '.ClusterLocalList')
+
+    def get_parser(self, prog_name):
+        parser = super(ClusterLocalList, self).get_parser(prog_name)
+        parser.add_argument(
+            'srv_types',
+            metavar='<srv_types>',
+            nargs='?',
+            help='Service Type(s)')
+        return parser
+
+    def take_action(self, parsed_args):
+        self.log.debug('take_action(%s)', parsed_args)
+        results = []
+        srv_types = parsed_args.srv_types
+        data = self.app.client_manager.admin.cluster_local_list()
+        for srv in data:
+            print srv
+            tags = srv['tags']
+            location = tags.get('tag.loc', 'n/a')
+            slots = tags.get('tag.slots', 'n/a')
+            volume = tags.get('tag.vol', 'n/a')
+            addr = srv['addr']
+            up = tags.get('tag.up', 'n/a')
+            score = srv['score']
+            srv_type = srv['type']
+            if not srv_types or srv_type in srv_types:
+                results.append((srv_type, addr, volume, location,
+                                slots, up, score))
+            columns = ('Type', 'Id', 'Volume', 'Location',
+                       'Slots', 'Up', 'Score')
+            result_gen = (r for r in results)
+        return columns, result_gen
+
+
+class ClusterUnlockService(command.Command):
+    """unlock score of the service given"""
```

Comment:
Don't speak about _the service given_, this is the role of the auto-generated usage.
And don't forget the capital letter.